src/main/scala/util/AsyncQueue.scala

```diff
@@ -96,10 +98,10 @@ class AsyncQueueSink[T <: Data](gen: T, depth: Int, sync: Int) extends Module {
   // This is safe, because isolation gates brought mem low before the zeroed widx reached us
   io.deq.bits  := RegEnable(io.mem(index), valid)
 
-  val valid_reg = AsyncResetReg(valid.asUInt)(0)
+  val valid_reg = AsyncResetReg(valid.asUInt, "valid_reg")(0)
```

Comment:
Or we should just add the missing form so you don't have to specify clock and reset, like the UInt form below.